modules/packages/Python.chpl

```diff
@@ -2217,6 +2217,157 @@ module Python {
     proc getPyObject() do return this.obj;
 
     // TODO: call should support kwargs
+
+    // Casts
+    /* Creates a new int from ``x``, when ``x`` is a :class:`Value`. */
+    operator :(x: borrowed Value, type t: int(?)): t throws {
+      var g = PyGILState_Ensure();
+      defer PyGILState_Release(g);
+
+      var pyObj = x.getPyObject();
+      Py_INCREF(pyObj);
+      var res = x.interpreter.fromPythonInner(t, pyObj);
+
+      return res;
```

Comment:
given that this code repeats in every single cast identically, can it be extracted to a helper function?